ios/Podfile

```diff
@@ -5,7 +5,7 @@ require File.join(File.dirname(`node --print "require.resolve('expo/package.json
 install! 'cocoapods',
          :generate_multiple_pod_projects => true,
          :incremental_installation => true
-platform :ios, '12.0'
+platform :ios, '13.0'
```

Comment:
good question. the correct version is 12.4 because of https://github.com/facebook/react-native/commit/982ca30de079d7e80bd0b50365d58b9048fb628f. i'll update accordingly.